modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,242 @@
++ 
+[source,yaml,subs="attributes+"]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline: 
+    bundles-resolver-config:
+      default-service-account: pipelines
+    cluster-resolver-config:
+      default-namespace: test
+    git-resolver-config:
+      server-url: localhost.com
+    hub-resolver-config: 
+      default-tekton-hub-catalog: tekton
+...
+----
+
+
+[id="chains-new-features-1-9_{context}"]
+=== {tekton-chains}
+
+:FeatureName: Tekton Chains
+include::snippets/technology-preview.adoc[]
+
+* Before this update, only Open Container Initiative (OCI) images were supported as outputs of `TaskRun` in in-toto provenance agent. This update adds in-toto provenance metadata as outputs with these suffixes, ARTIFACT_URI and ARTIFACT_DIGEST.
+
+* Before this update, only `TaskRun` attestations were supported. This update adds support for `PipelineRun` attestations as well. 
+
+* This update adds support for {tekton-chains} to get the `imgPullSecret` parameter from the pod template. This update helps you to configure repository authentication based on each pipeline run or task run without modifying the service account.
+
+[id="tekton-hub-new-features-1-9_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* With this update, as an administrator, you can use an external database, such as Crunchy PostgreSQL with {tekton-hub}, instead of using the default {tekton-hub} database. This update helps you to perform the following actions:
+** Specify the coordinates of an external database to be used with {tekton-hub}
+** Disable the default {tekton-hub} database deployed by the Operator
+
+* This update removes the dependency of `config.yaml` from external Git repositories and moves the complete configuration data into the API `ConfigMap`. This update helps an administrator to perform the following actions:
+** Add the configuration data, such as categories, catalogs, scopes and defaultScopes in the {tekton-hub} custom resource.
+** Modify {tekton-hub} configuration data on the cluster. All modifications are preserved upon Operator upgrades.
+** Update the list of catalogs for {tekton-hub}
+** Change the categories for {tekton-hub}
++ 
+[NOTE]
+====
+If you do not add any configuration data, you can use the default data in the API `ConfigMap` for {tekton-hub} configurations.
+====
+
+[id="pac-new-features-1-9_{context}"]
+=== {pac}
+
+* This update adds support for concurrency limit in the `Repository` CRD to define the maximum number of `PipelineRuns` running for a repository at a time. The `PipelineRuns` from a pull request or a push event are queued in alphabetical order.
+
+* This update adds a new command `tkn pac logs` for showing the logs of the latest pipeline run for a repository.
+
+* With this update, you can modify {pac} configurations in the `TektonConfig` CRD.
```

Comment:
This should be part of Operator section i feel :thinking: because PAC doesn't have any `TektonConfig` CRD 
cc @chmouel @sm43

Reply:
So adding it to Operator section just after this https://github.com/openshift/openshift-docs/pull/54491/files#diff-47dac456c0d7ba50140f46956d4d6d40b4b47084258f26de0646210263a6227fR84 would be more clear

Reply:
@savitaashture Incorporated your suggestions. Please have a look at the updated PR.